pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/DefaultSchemaRegistryService.java

```diff
@@ -71,17 +75,17 @@
     }
 
     @Override
-    public CompletableFuture<SchemaVersion> deleteSchema(String schemaId, String user, boolean force) {
+    public CompletableFuture<SchemaVersion> putEmptySchema(String schemaId, String user, boolean force) {
```

Comment:
Why do we need to change it to `putEmptySchema`?

Reply:
this method only appends an empty schema. the existing `deleteSchemaStorage(...)` method actually delete's the schema from the storage. this rename is so that the method name will reflect the behavior.